common/changes/@uifabric/experiments/inateeg-0611list_2019-06-26-19-24.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@uifabric/experiments",
```

Comment:
This change file shouldn't be needed, see https://github.com/OfficeDev/office-ui-fabric-react/pull/9432/files#diff-a4c06458fab060a6f0be01f6290a290a